features/apim-analytics-feature/org.wso2.analytics.solutions.apim.analytics.feature/src/main/resources/data-purging-siddhi-file/APIM_ANALYTICS_DATA_PURGING.siddhi

```diff
@@ -0,0 +1,26 @@
+@App:name("APIM_ANALYTICS_DATA_PURGING")
+@App:description("App purges data of Raw tables in APIM Analytics")
+-- Please refer to http://wso2.github.io/siddhi/documentation/siddhi-4.0/
+
+define trigger ThreeMonthTrigger at every 3 months;
+define stream MinTriggerEventStream (triggered_time long);
+
+-- Table Used in Unusual IP Access Alert
+@PrimaryKey('applicationConsumerKey','ip')
+@store(type = 'rdbms', datasource = 'APIM_ANALYTICS_DB')
+define table ApimIPAccessSummary (username string, applicationConsumerKey string, ip string, lastAccessedDate long) ;
+
+-- Table Used in Unusual IP Access Alert
+@PrimaryKey('applicationConsumerKey','username')
+@store(type = 'rdbms', datasource = 'APIM_ANALYTICS_DB')
+define table ApimIPAccessAlertCount (username string, applicationConsumerKey string, requestCount long) ;
+
+-- Delete all Data from ApimIPAccessSummary when the trigger is triggered
+from ThreeMonthTrigger
+delete ApimIPAccessSummary on TRUE;
```

Comment:
Please change on condition, this siddhi query will delete 'ALL'  records when it is triggered
from Trigger
delete ApimIPAccessSummary 
on timestamp <= triggered_time - '<User configured value>';

Reply:
This file has been removed form PR